dts/riscv/it8xxx2.dtsi

```diff
@@ -1416,9 +1416,8 @@
 				   &wuc_wu35   /* KSI[5] */
 				   &wuc_wu36   /* KSI[6] */
 				   &wuc_wu37>; /* KSI[7] */
-			gpio-dev = <&gpioc>;
-			pinctrl-0 = <&pinctrl_kso16   /* GPC3 */
-				     &pinctrl_kso17>; /* GPC5 */
+			kso16-gpios = <&gpioc 3 0>;
+			kso17-gpios = <&gpioc 5 0>;
```

Comment:
see my previous comment about flags. are kso16/17 only routable to GPC3/5?

Reply:
Done. Yes.